examples/tox21/keras_fcnet.py

```diff
@@ -0,0 +1,60 @@
+"""
+Script that trains multitask models on Tox21 dataset.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import numpy as np
+import keras
+from keras import Input, Model
+from keras.layers import Dense, Dropout, Multiply
```

Comment:
These should be tf.keras.

Reply:
yes it should -- I did this one first before the GC one